docs/pages/docs/features.en.mdx

```diff
@@ -8,7 +8,9 @@ import { Callout } from 'nextra-theme-docs'
 # Features
 
 <Callout>
-  The following use abbreviated examples that skip a complete setup of passing Nexus type definition to Nexus' `makeSchema`. If you are new to Nexus, consider reading the [official Nexus tutorial](https://nxs.li/tutorial) before jumping into Nexus Prisma.
+  The following use abbreviated examples that skip a complete setup of passing Nexus type definition to Nexus'
+  `makeSchema`. If you are new to Nexus, consider reading the [official Nexus
+  tutorial](https://nxs.li/tutorial) before jumping into Nexus Prisma.
```

Comment:
These spaces look like a mistake.

Reply:
The file was formatted with the existing prettier settings hence the spacing. But there is no visible difference in the docs.

Reply:
If you're saying this is how prettier formats then fine.